src/tests/ftest/io/ior_intercept_dfuse_mix.py

```diff
@@ -0,0 +1,239 @@
+        self.assertTrue(float(with_intercept[2][0][min_mib]) <
+                        write_x * float(without_intercept[2][0][min_mib]))
+        self.assertTrue(float(with_intercept[2][0][mean_mib]) <
+                        write_x * float(without_intercept[2][0][mean_mib]))
+
+        # Verifying read performance
+        # Read performance is not significant with interception library
+        # and most likely the read_x will be 1. To avoid unnecessary
+        # failure keeping flat 1.5 x just to set the boundary for the client
+        # without interception library
+        self.assertTrue(float(with_intercept[2][1][max_mib]) <
+                        1.5 * float(without_intercept[2][1][max_mib]))
+        self.assertTrue(float(with_intercept[2][1][min_mib]) <
+                        1.5 * float(without_intercept[2][1][min_mib]))
+        self.assertTrue(float(with_intercept[2][1][mean_mib]) <
+                        1.5 * float(without_intercept[2][1][mean_mib]))
+
+    def log_metrics(self, without_intercept, with_intercept):
+        """Log the ior metrics because the stdout from ior can be mixed
+           because of multithreading.
+
+           Args:
+               without_intercept (dict): IOR Metrics without using
+                                         interception library.
+               with_intercept (dict): IOR Metrics using interception
+                                      library.
+        """
+        IorCommand.log_metrics(self.log, "3 clients - without " +
+                               "interception library", without_intercept[1])
+        IorCommand.log_metrics(self.log, "3 clients - with " +
+                               "interception library", with_intercept[1])
+        IorCommand.log_metrics(self.log, "1 client - without " +
+                               "interception library", without_intercept[2])
+        IorCommand.log_metrics(self.log, "1 clients - without " +
+                               "interception library", with_intercept[2])
+
+    def run_multiple_ior_with_pool(self, results, intercept=None):
+        """Execute ior with optional overrides for ior flags and object_class.
+
+        If specified the ior flags and ior daos object class parameters will
+        override the values read from the yaml file.
+
+        Args:
+            intercept (str): path to the interception library. Shall be used
+                             only for POSIX through DFUSE.
+            ior_flags (str, optional): ior flags. Defaults to None.
+            object_class (str, optional): daos object class. Defaults to None.
+        """
+        # Create a pool if one does not already exist
+        if self.pool is None:
+            self.create_pool()
+        # Update IOR params with the pool
+        self.ior_cmd.set_daos_params(self.server_group, self.pool)
+
+        # start dfuse if api is POSIX
+        if self.ior_cmd.api.value == "POSIX":
```

Comment:
I don't think you need this as you are only using POSIX api.

Reply:
Done.